run_tests.R

```diff
@@ -20,11 +20,11 @@ allTests <- row.names(testTimes)
 
 # Run each test in a separate process to avoid dll garbage overload.
 for (test in allTests) {
-    runViaTestthat <- FALSE  # TODO Fix test-size.R and set this to TRUE.
+    runViaTestthat <- TRUE  # TODO Fix test-size.R and set this to TRUE.
```

Comment:
Did you intend to set `runViaTestthat <- TRUE` in the PR or save flipping that switch for later?